app/views/events/index.html.erb

```diff
@@ -13,23 +13,14 @@
   <fieldset class="usa-fieldset">
     <legend class="usa-legend"><h3>Filter by audience</h3></legend>
     <% @audiences.each do |audience| %>
-      <%
-        is_selected = @selected_audiences.include?(audience)
-        is_only_one_selected = is_selected && @selected_audiences.size == 1
-      %>
-      <% if is_only_one_selected then %>
-        <% # We're disabling the checkbox, but we need to persist the value on form submit %>
-        <input type="hidden" name="audience[]" value="<%= audience %>">
-      <% end %>
       <div class="usa-checkbox">
         <input
-          class="usa-checkbox__input usa-checkbox__input--tile"
+          class="usa-checkbox__input"
           id="filter-<%= audience.parameterize %>"
           type="checkbox"
           name="audience[]"
           value="<%= audience %>"
-          <%= if is_selected then "checked" end %>
-          <%= if is_only_one_selected then "disabled" end %>
+          <%= if is_selected = @selected_audiences.include?(audience) then "checked" end %>
```

Comment:
[suggestion] we don't need to set the variable here anymore, since it isn't read anywhere

```suggestion
          <%= if @selected_audiences.include?(audience) then "checked" end %>
```